php_phongo.c

```diff
@@ -2016,6 +2016,18 @@ static void php_phongo_client_dtor(void *client)
 }
 #endif
 
+static void _phongo_check_json (void) __attribute__((constructor));
+static void _phongo_check_json (void)
```

Comment:
PHP's mechanism is as follows:

- ``dlopen()`` all extensions in their loading order (order as in .ini file). This works normally, as ``LTDL_LAZY`` is being used and no symbols are being resolved yet.
- create dependency graph
- go initialize extensions in the right order as to not have undefined symbols

The new constructor is run on ``dlopen()``, and hence, breaks the original mechanism of being to *open* extensions in a random order.